depot/containerstore/containerstore.go

```diff
@@ -372,3 +375,7 @@ func (cs *containerStore) NewRegistryPruner(logger lager.Logger) ifrit.Runner {
 func (cs *containerStore) NewContainerReaper(logger lager.Logger) ifrit.Runner {
 	return newContainerReaper(logger, &cs.containerConfig, cs.clock, cs.containers, cs.gardenClient, cs.reapingLock)
 }
+
+func calculateEntitlement(memoryLimit, containerAge uint64) uint64 {
+	return (memoryLimit * containerAge) / (1024 * 1024 * 1024)
```

Comment:
I don't understand why this is the correct formula for the CPU entitlement to date, and the corresponding tests here don't illuminate this formula. Could you change both the code and the tests to explain? Named constants instead of inline numbers may also help.

Reply:
I was originally going to ask why emit something that is proportional to memory when we can emit some constant value ? After reading the story linked to in the previous comment, I am more confused. It sounds like we are already trying to integrate those metrics with the cli when they aren't ready yet ?!

Reply:
If we emit a constant, that would not meet the acceptance criteria in the Garden story:
```
4 See that if I double the memory limit the calculated CPU usage would be half as much as before
```

> It sounds like we are already trying to integrate those metrics with the cli when they aren't ready yet
No, we are not integrating these numbers with CLI yet, first we want to make the numbers available to the nozzle and we need to start from somewhere. It would be a huge story to implement the configuration, the calculations and sending all that to the nozzle, hence we decided to split these into separate smaller stories. Makes sense?

Reply:
As per using inline numbers - 1024 is a pretty popular constant when it comes to converting memory size. Sill, I introduced the [`gigabytes`](https://github.com/cloudfoundry/executor/pull/35/commits/1c23cbd8d1f0bbba2960d850dbff1c7a5e0aa821#diff-4317b4ae71cea25c1130819c69e18007R383) utility function, I hope that it makes the code more clear.

Reply:
I think @jvshahid and I disagree with the premise of that part of the acceptance criteria. It seems undesirable to us to have the rep emit a blatantly incorrect number for this metric solely to enable end-to-end integration of the CLI plugin. Especially since no changes to the loggregator type system are required at all now, I don't think it would be too onerous to leave it out for now and then to add the expected entitlement metric only after we have a reasonable first approximation to a meaningful value.